### ISSUE TITLE
Incorrect typings of Snackbar positions

### ISSUE THREAD

**Issue by paradis-A** [illright/attractions#256]
![Screenshot from 2021-05-02 10-21-20](https://user-images.githubusercontent.com/42695235/116800069-4b988a00-ab30-11eb-9ecc-c52ccc8cd699.png)
Im trying to change position but im not able to.

Error message:
Type 'string' is not assignable to type 'Readonly<{ TOP_LEFT: string; TOP_MIDDLE: string; TOP_RIGHT: string; MIDDLE_RIGHT: string; BOTTOM_RIGHT: string; BOTTOM_MIDDLE: string; BOTTOM_LEFT: string; MIDDLE_LEFT: string; }>'.

Please move to this to discussion if this invalid issue.

**Comment by illright** [comment]
I'm unable to diagnose the problem from what you've sent. Could you please share the rest of the code (ideally, as code-formatted text and removing as much code as possible such that the issue still persists)?

**Comment by paradis-A** [comment]
```javascript
// example.svelte
<script lang="ts">
  import { SnackbarContainer, Button } from "attractions"
  import { SnackbarPositions } from "attractions/snackbar"
  let snackbar
  let position = SnackbarPositions.BOTTOM_LEFT
  const handleClick = () => {
    snackbar.showSnackbar({ props: { text: "Hello!" } })
  }
</script>

<Button on:click={handleClick} />
<SnackbarContainer {position} bind:this={snackbar} />
```

Problem:
Type 'string' is not assignable to type 'Readonly<{ TOP_LEFT: string; TOP_MIDDLE: string; TOP_RIGHT: string; MIDDLE_RIGHT: string; BOTTOM_RIGHT: string; BOTTOM_MIDDLE: string; BOTTOM_LEFT: string; MIDDLE_LEFT: string; }>'.

Probably typescript support? It actually works without lang="ts" on the script tag.

**Comment by aabounegm** [comment]
@paradis-A Yes, it was indeed a TypeScript problem from our side. Your code is totally correct, but we had the wrong typings. It should be fixed now with the `v3.2.1` update.
